com.unity.shadergraph/Documentation~/Gradient-Noise-Node.md

```diff
@@ -12,6 +14,13 @@ Generates a gradient, or [Perlin](https://en.wikipedia.org/wiki/Perlin_noise), n
 | Scale      | Input | Float    | None | Noise scale |
 | Out | Output      |    Float    | None | Output value |
 
+## Controls
+
+| Name        | Type           | Options  | Description |
+|:------------ |:-------------|:-----|:---|
+| Hash Type      | Dropdown | Deterministic, LegacyMod | Selects the hash function used to generate random numbers for noise generation. |
```

Comment:
Should this say "LegacyMode" (rather than LegacyMod)?

Reply:
it is actually a modulo operation.